src/App/components/Section/Section.tsx

```diff
@@ -103,56 +131,12 @@ const Section: React.FC = () => {
           />
         );
       });
+    } else {
+      return <EmptyState />;
     }
```

Comment:
in first condition we are returning the value and it is not needed else here,
```
if (true) return a;
return b;
```
not need else

Reply:
I have done this now